XenModel/XenAPI/JsonRpcClient.cs

```diff
@@ -12205,6 +12205,20 @@ public void message_destroy(string session, string _message)
             Rpc("message.destroy", new JArray(session, _message ?? ""), serializer);
         }
 
+        public void message_destroy_many(string session, List<XenRef<Message>> _messages)
+        {
+            var converters = new List<JsonConverter> {new XenRefListConverter<Message>()};
+            var serializer = CreateSerializer(converters);
+            Rpc("message.destroy_many", new JArray(session, _messages == null ? new JArray() : JArray.FromObject(_messages, serializer)), serializer);
+        }

```

Comment:
Do we need to add the non-async version? It should be added the next time we update the SDK, anyway

Reply:
I just added both because when I was first trying out the new call I was using the sync version, but then I decided to switch to the async one (so the session is not blocked in case xapi takes too long to dismiss a huge number of alerts on a real system).